models/app.models.js

```diff
@@ -15,3 +15,15 @@ exports.fetchArticleById = (articleID) => {
         return rows[0];
     })
 }
+
+exports.fetchCommentsByArticleId = (articleId) => {
+    return db.query(`SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id FROM comments
+    LEFT JOIN articles
+    ON comments.article_id = articles.article_id
+    WHERE articles.article_id = $1
+    ORDER BY comments.created_at DESC;`, [articleId])
+    .then(({rows}) => {
+        return rows;
+    })
```

Comment:
When testing for a valid ID but no comments, you will need to amend your model to check that the article exists but has no comments. You will first need to check whether the article exists in order to determine this 👀